README.md

```diff
@@ -14,20 +14,26 @@
 </p>
 
 ## About The Project
-This project is part of Turing Software and Designs' Front-End Program, Module 3. The project focuses on creating an application that utilizes React to create an application that allows user's to manage and rate different movies. The program is implemented with ES6 syntax. This program focuses on working with an API to send and receive data
+This project is part of Turing Software and Designs' Front-End Program, Module 3. The project focuses on creating an application that utilizes React to allows users to manage and rate different movies. The program is implemented with ES6 syntax and uses CSS for styling and HTML through JSX syntax. This program focuses on working with an API to send and receive data. The data is retrieved and displayed in various paths throughout the platform. React Router is utilized for path transitions and displays. Each component has a robust Jest testing suite to account for both happy and sad path testing as well as unit and integration testing.
 
 
 ### Built With
```

Comment:
I like that you updated the README with the technologies that we're using. This will make final edits easy & quick